index.html

```diff
@@ -1118,11 +1093,11 @@
           <p class="note">
```

Comment:
A few lines above this there's a `<a>shippingOption</a>` that should be `<a data-lt="PaymentRequest.shippingOption">shippingOption</a>` I think. Preexisting problem maybe?